lib/passport-azure-ad/wsfedstrategy.js

```diff
@@ -51,7 +53,7 @@ function Strategy (options, verify) {
   this.certs = [];
 
   // Create the metadata object if the user has specified a federation metadata url
-  if(options.identityMetadata) {
+  if (options.identityMetadata) {
     this.metadata = new Metadata(options.identityMetadata);
```

Comment:
if possible it may read better to rename options.identityMetadata -> options.metadata